config/config.edn

```diff
@@ -4,4 +4,8 @@
                     :dev #ref [:secrets :dev-db]
                     :prod #ref [:secrets :prod-db]
                     :ci #ref [:secrets :ci-db]}
+ :log #profile {:test {}
```

Comment:
If there is no difference in the config, what's the use of using #profile? 

Can you add the appropriate config values?

Reply:
I added #profile in case we need to modify the config in different scenarios but the default config provided by `timbre` works for our usecase right now.
I will remove #profile & keep an empty log config for now